scripts/plugins/caching-config.js

```diff
@@ -0,0 +1,23 @@
+export const Config = {
+  headers: { "hasura-m-auth": "zZkhKqFjqXR4g5MZCsJUZCnhCcoPyZ" },
+  time_to_live: 600,
+  redis_url: "redis://redis:6379",
```

Comment:
The hostname won't be redis right? Because we're running axiom on Hasura cloud and redis will be somewhere else e.g. on AWS. Would hostname be best as an environment variable too? (Especially given we have geographically distributed demo instances)

Reply:
ok read this in more detail. This can be redis: but the other reference to port 8787 where the cache worker/plugin runs should be pointed to a hostname